### PR TITLE
Added html img component instad of next/image when using srcSet

### PR DESCRIPTION
**Pull request by luiztiago**
## Summary

- Added the `` component instead of next/image when using `srcSet`
- It has been added on PostContent file, to be used on Group Blocks.

## Notes

- Added the `@next/next/no-img-element` on the eslint rules, because was getting an error to don't use the `` tag.
- `components/PostContent/PostContent.tsx` line 27 was added to prevent warning on if statement.

### CODE REVIEW

**Review by elazzabi — APPROVED** [review]
I think this looks good, but I have a question.

So it looks like the presence of `srcSet` in the props is what decides if we'll use `` or `Image`. However, from our WP bundle, it looks like the `srcSet` attribute [will always be there](https://github.com/Automattic/vip-decoupled-bundle/blob/5f31805fa96746f1853fd781308d6b13f57eaed0/blocks/blocks.php#L94) and [we ask for all attributes by default](https://github.com/Automattic/vip-go-nextjs-skeleton/blob/trunk/graphql/fragments/ContentNode.graphql#L9). So, in theory, we'll always use `` 🤔 . If that's true, I guess we should document it.

**Comment by luiztiago** [comment]
@elazzabi Make sense, I did some tests here, and you're right. So maybe we can add a config inside the `next.config.js` file and check it inside the if statement. What do you think?

**Comment by luiztiago** [comment]
@elazzabi I did the update with the config approach. Let me know what do you think about it.

File: components/PostContent/PostContent.tsx
Comment (luiztiago) [inline components/PostContent/PostContent.tsx]: I added the `config` in lowercase because is the pattern on all files importing `next.config.js` file

File: components/PostContent/PostContent.tsx
Comment (elazzabi) [inline components/PostContent/PostContent.tsx]:
```suggestion
								srcSet: blockProps.srcset || undefined,
```

Looks like the attribute we return from WP is `srcset` instead of `srcSet`

File: components/PostContent/PostContent.tsx
Comment (elazzabi) [inline components/PostContent/PostContent.tsx]:
```suggestion
									<img alt={blockProps.alt} {...imageProps} />
```

This should pass `imageProps` in order for `srcSet`, `src`, `width` and `height` to be populated

**Review by elazzabi — CHANGES_REQUESTED** [review]
I changed a few things locally to make it work, I posted them as suggestions.

I think the config approach is good to make this configurable by customers 👍

**Comment by luiztiago** [comment]
@elazzabi Talking with @chriszarate today, we had the idea to create a new file with the vip.configs to keep the `next.config.js` file with only the settings from next.js. I moved the logic and comments to the new file and worked on your requests. Thanks :)

**Review by elazzabi — APPROVED** [review]
> create a new file with the vip.configs to keep the next.config.js file with only the settings from next.js

Sounds like a good idea!

This should be good to go after merging the last suggested change ^

**Comment by luiztiago** [comment]
@elazzabi Can you double-check the last suggested change? If I understood your request, it's already been done.

**Comment by elazzabi** [comment]
@luiztiago Hmm weird, it still shows that we are using `defaultProps` instead of `imageProps` 🤔

**Comment by luiztiago** [comment]
@elazzabi It's fixed now, sorry I misunderstood your request before.

**Comment by luiztiago** [comment]
@elazzabi Hey, just a reminder, if you can approve it again because I did some updates after your first approval :)

**Review by chriszarate — APPROVED** [review]

**Pull request merged**
